coalib/settings/Setting.py

```diff
@@ -53,8 +53,17 @@ def typed_list(conversion_func):
                             your desired list item object.
     :return:                A conversion function.
     """
-    return lambda setting: [
-        conversion_func(StringConverter(elem)) for elem in setting]
+
+    class Converter:
+
+        def __call__(self, setting):
+            return [conversion_func(StringConverter(elem))
+                    for elem in setting]
+
+        def __repr__(self):
+            return 'typed_list(%s)' % conversion_func.__name__
```

Comment:
@Makman2 Just try it ;)

`__repr__` has to defined in class, but `lambda` is a `function` instance, and you can't derive the `function` class:

`TypeError: type 'function' is not an acceptable base type`